shared/spikeratestats_Events_Baseline.m

```diff
@@ -0,0 +1,739 @@
+function [stats] = spikeratestats_Events_Baseline(cfg,force)
```

Comment:
Nice, but as a function it does too much and is also too specific at the same time to be a useful 'modular' function in /shared I think. I would suggest to move it to your project/scripts directory, where other people can use/try it if they want, and see if it can be of generic use.